### PR TITLE
Update job fail message

### PR DESCRIPTION
**Pull request by allenwq**
The old message is misleading, and student might think it's their faults.

### CODE REVIEW

File: config/locales/en/course/assessment/answer/answers.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ en:
   course:
     assessment:
       answer:
-        error: 'An error occurred while evaluating your answer, please try again.'
+        error: >
+          Sorry. Looks like something went wrong on our end, please try again.
```
Comment (fonglh) [inline config/locales/en/course/assessment/answer/answers.yml]: "Sorry, something went wrong on the code evaluator. Please try again."

Comment (allenwq) [inline config/locales/en/course/assessment/answer/answers.yml]: Should we more general ? cause might be a time out.

Comment (allenwq) [inline config/locales/en/course/assessment/answer/answers.yml]: Oh wait, it's not only code evaluator, MCQ also shares the same message.

Comment (fonglh) [inline config/locales/en/course/assessment/answer/answers.yml]: Ohh. "Sorry, the autograder is having mood swings and quit on us. Please try again."

Comment (allenwq) [inline config/locales/en/course/assessment/answer/answers.yml]: Ok, used your msg :-P

**Pull request merged**